benches/benches.rs

```diff
@@ -23,50 +24,57 @@ extern crate test;
 use noise::{perlin2, perlin3, perlin4, simplex2, simplex3, simplectic2, simplectic3, simplectic4, Seed};
 use test::Bencher;
 
+fn black_box<T>(dummy: T) -> T {
```

Comment:
It's private and has a comment on the inside.

On Sun, Jan 4, 2015 at 1:49 AM, Brendan Zabarauskas <
notifications@github.com> wrote:

> In benches/benches.rs
> <https://github.com/bjz/noise-rs/pull/44#discussion-diff-22437153>:
>
> > @@ -23,50 +24,57 @@ extern crate test;
> >  use noise::{perlin2, perlin3, perlin4, simplex2, simplex3, simplectic2, simplectic3, simplectic4, Seed};
> >  use test::Bencher;
> >
> > +fn black_box<T>(dummy: T) -> T {
>
> Would be nice to have a comment on this
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bjz/noise-rs/pull/44/files#r22437153>.
>



-- 
http://octayn.net/